### ISSUE TITLE
Improvements to the vignette

### ISSUE THREAD

**Issue by wleoncio** [SystemsBiologist/CellMigRation#34]
Source: https://github.com/Bioconductor/Contributions/issues/1618#issuecomment-696258765

- [x] REQUIRED: The Installation section should demonstrate to the user how to
    install the package from Bioconductor, not github. The code should look like
    the following:

```
if(!requireNamespace("BiocManager", quietly = TRUE))
    install.packages("BiocManager")
BiocManager::install("cellmigRation")
```

- [x] SUGGESTION: We strongly encourage a table of contents.

- [x] REQUIRED: Lines 114-126, we discourage the use of downloading data from
online resources and would much rather have the data as part of the package. If
you want to demonstrate LoadTiff() I would suggest including these data files
as part of an inst/extdata/ directory and include the download process as a
script in an inst/script/ directory.

- [x] REQUIRED: You need to show the loading of the LS data
(data("VignBuilderDataset")). I would suggest explaining that the loading in
the tiff data is optional (and can be done with the users own data) but the
data can be loaded for the vignette using data(VignBuilderDataset). That way
if the user just wants to walk through the vignette they won't have to download
the data themselves, they can just load yours and be on their way.

- [x] REQUIRED: Code chunks need to be evaluated (eval = TRUE) and visible to
the user (echo = TRUE), unless there is some justification as to why they
shouldn't be. The idea behind a vignette is to have reproducibility and if
chunks of code aren't evaluated or shown it can result in code that either
won't work for the user or different results.

- [x] REQUIRED: Line 387, there seems to be an extra ` before CellMig().

**Comment by wleoncio** [comment]
I would add a suggestion to reduce the vignette by 10%, so that the doc folder is under 1 MB and `devtools::check()` stops complaining.

**Comment by wleoncio** [comment]
Addressed on f38498fdda3554d5ef33613d1b5ee4970b94bdc2.
